Basic FE project template/index.html

```diff
@@ -11,10 +11,56 @@
 
   <body>
     <main>
-      <div>pateu gras</div>
-      <div>
-        <img src="ui-src/assets/images/audicar.webp" alt="car" />
-      </div>
+      <section class="why-beleasing">
+        <div class="container">
+          <div class="why-beleasing__title">
+            <h1>WHY BELEASING?</h1>
+          </div>
+          <div class="why-beleasing__description">
+            <p>
+              Nullam suspendisse mauris blandit cum pharetra odio mauris lacus
+              risus. Velit at nam venenatis nunc lorem aliquam.
+            </p>
+          </div>
+          <ul class="advantages">
+            <li class="advantages__item">
+              <img
+                src="ui-src/assets/images/beleasing2-home-icon1.svg"
```

Comment:
change the src "ui-src/assets/images/beleasing2-home-icon1.svg" to "ui/assets/images/beleasing2-home-icon1.svg". The "ui' folder is the live folder, the "ui-src" folder is for development purposes.